net-im/skypeforlinux/skypeforlinux-5.1.0.1.ebuild

```diff
@@ -18,36 +20,36 @@ S="${WORKDIR}"
 QA_PREBUILT=opt/skypeforlinux/skypeforlinux
 RESTRICT="mirror bindist strip" #299368
 
-RDEPEND="dev-libs/atk
-	dev-libs/expat
-	dev-libs/glib:2
-	dev-libs/nspr
-	dev-libs/nss
-	gnome-base/gconf:2
+RDEPEND="dev-libs/atk[${MULTILIB_USEDEP}]
+	dev-libs/expat[${MULTILIB_USEDEP}]
+	dev-libs/glib:2[${MULTILIB_USEDEP}]
+	dev-libs/nspr[${MULTILIB_USEDEP}]
+	dev-libs/nss[${MULTILIB_USEDEP}]
+	gnome-base/gconf:2[${MULTILIB_USEDEP}]
 	gnome-base/libgnome-keyring
```

Comment:
It looks like gnome-base/libgnome-keyring isn't linked to any of gnome-base/gnome-keyring's libraries. It just starts gnome-keyring-daemon which works in a manner similar to ssh-agent (sockets in /var/run/user/$UID/keyring/), so it actually don't have to be 64-bit. I will try to make libgnome-keyring multilib-ed first and take care of gnome-keyring next. Should I make another PR or just add commits to this one?